examples/gnrc_networking/Makefile

```diff
@@ -29,6 +29,8 @@ USEMODULE += gnrc_icmpv6_echo
 USEMODULE += shell
 USEMODULE += shell_commands
 USEMODULE += ps
+USEMODULE += netstats_l2
+USEMODULE += netstats_ipv6
```

Comment:
`DISABLE_MODULE=netstats_ipv6`. But it was just food for thought. I am fine with both variants.